tests/test_bug_features.py

```diff
@@ -38,15 +39,26 @@
 
 @pytest.fixture
 def read(get_fixture_path):
-    def _read(path, feature_extractor_class, expected_results):
+    def _read(
+        path,
+        feature_extractor_class,
+        expected_results,
+        use_inline_data=False,
+        inline_data="",
```

Comment:
I would avoid having empty strings as the default value. It will be more explicit to use `None`.

Reply:
0197f24e5750411eeca1d52df8f3c2f12f07270e